tools/manifest-templator/manifest-templator.go

```diff
@@ -260,7 +260,7 @@ func getCDI(data *templateData) {
 	// Get CDI Deployment
 	cdideployment := cdicomponents.GetDeployment(
```

Comment:
We need to drop cdicomponents in favor of https://github.com/kubevirt/containerized-data-importer/blob/master/pkg/operator/resources/operator/components.go

If you are comfortable making that change feel free, otherwise I'll create a follow-up PR.